src/Dropdown/Dropdown.js

```diff
@@ -1,11 +1,18 @@
-import React, { Fragment } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from 'styled-components';
 import Popper from 'popper.js';
 import Box from '../Box';
 import Portal from '../Portal';
 import { createComponent, themeGet } from '../utils';
 
+const DropdownWrapper = createComponent({
+  name: 'DropdownWrapper',
+  style: css`
+    display: inline-block;
```

Comment:
This could be considered a breaking change so let's think through if this could have unintended consequences anywhere we currently use Dropdown

Reply:
Due to the event configuration being removed (on line 47, etc.)?

Reply:
If so, I could not find any instances of the `on` prop being used to specify an alternative event listener.

Reply:
Due to adding the new `inline-block` style to the wrapper, Dropdown now becomes inline with their previous sibling, which may have unintended consequences anywhere we use the component.

## Before change
![screen shot 2019-03-01 at 9 43 24 am](https://user-images.githubusercontent.com/14184138/53655923-7dd39780-3c06-11e9-8f13-cf740a58fba6.png)

## After change
![screen shot 2019-03-01 at 9 43 29 am](https://user-images.githubusercontent.com/14184138/53655928-7f04c480-3c06-11e9-857e-b57830daea5d.png)

---

Regarding `on` change, we have a ticket coming up that will require showing the dropdown on hover. I'd prefer not to move or if we need to make changes to support accessibly in this ticket, let's do that in this PR: https://app.asana.com/0/1107902406623186/1112247428608998

Reply:
Many thanks @kylealwyn, will remove the styling change, and revert the removal of the event configuration.

Reply:
In order to support accessibility, we would likely need to switch from popper.js to something like downshift, as discussed [here](https://app.asana.com/0/821652619613450/1110934258569822/f).

Reply:
@kylealwyn do we still want to support hover if the upcoming ticket has changed to be click-based?